extension.neon

```diff
@@ -233,6 +235,9 @@ parameters:
 parametersSchema:
 	drupal: structure([
 		drupal_root: string()
+		bleedingEdge: structure([
+			checkDeprecatedHooks: boolean()
+		])
```

Comment:
I want to rename this about "loading API files to check for deprecated hooks" so we can keep the rule and enable configured deprecated hooks manually